scripts/format-code

```diff
@@ -159,6 +159,7 @@ check_clang-format()
             echo "clang-format is not installed. Please run scripts/install-prereqs first."
```

Comment:
Above at line 151 there is an old comment:

> # First check explicitly for the 3.8 executable.

That should be updated for Clang 7.